### ISSUE TITLE
Netconf confirmed commit

### ISSUE THREAD

**Issue by sasko-tanaskoski** [PANTHEONtech/lighty#987]
Is confirmed commit supported in lighty or ODL? If so, some example maybe. Thanks.

**Comment by sasko-tanaskoski** [comment]
Hi, I thought on NETCONF, confirmed commit described in RFC-6241 8.4, or in other words network-wide transaction on multiple devices.

Regards

**Comment by ihrasko** [comment]
No. We are sorry. This feature is not supported in ODL nor Lighty.io.
